src/clj/nnts2/model/directory.clj

```diff
@@ -16,10 +17,12 @@
         next-iter-rows (clojure.set/difference (set remaining-rows) (set to-insert))]
     (reduce
      (fn [acc v]
```

Comment:
Can we rename the `v` to be the element type in question? Like "directory"?

Can destructure: `id, name, org-id` in the function param itself.